sholes/roster.json

```diff
@@ -24,7 +24,13 @@
       },
       {
         "name": "Abdul Kamara",
-        "github": "abdul-san"
+        "github": "abdul-san",
+        "materials": [
+          {
+            "type2": "cv",
+            "type": "video",
+            "url": "https://youtu.be/e80xdyx5g-o"
+          }      ]
```

Comment:
Please have a look at the instructions :)

Reply:
I made another commit with the updated id.